plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageActionAspect.java

```diff
@@ -105,56 +98,48 @@ public void executeExtraValueFormForEdit(JoinPoint joinPoint) {
             SeoPageMetadata pageMetadata = (SeoPageMetadata) page.getMetadata();
             request.setAttribute(PARAM_FRIENDLY_CODE, pageMetadata.getFriendlyCode());
             request.setAttribute(PARAM_USE_EXTRA_DESCRIPTIONS, pageMetadata.isUseExtraDescriptions());
-            request.setAttribute(PARAM_XML_CONFIG, pageMetadata.getXmlConfig());
             ApsProperties props = pageMetadata.getDescriptions();
             if (null != props) {
-                Iterator<Object> it = props.keySet().iterator();
-                while (it.hasNext()) {
-                    String key = (String) it.next();
-                    request.setAttribute(PARAM_DESCRIPTION_PREFIX + key, props.get(key));
+                Iterator<Object> iter = props.keySet().iterator();
```

Comment:
Does this have test coverage?

Reply:
In test class TestPageAction (in test package) I have tested also the aspect class